### PR TITLE
Right margin wrapping

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
We need to wrap all code to 100 column width.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageShould.java [hunk 1]
```diff
@@ -65,14 +65,16 @@ public static void tearDownClass() {
         datastoreFactory.tearDown();
     }
 
-    @SuppressWarnings("ConstantConditions") // passing null because this parameter isn't used in this implementation
+    @SuppressWarnings("ConstantConditions")
+    // passing null because this parameter isn't used in this implementation
```
Comment (dmdashenkov) [inline gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageShould.java]: Please have a tabulation (4 spaces) at the start of the line. This is required to make it obvious that the comment belongs to this specific suppression.

File: gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageStatusHandlingShould.java [hunk 2]
```diff
@@ -33,7 +33,8 @@
 /**
  * @author Dmytro Dashenkov.
  */
-public class DsAggregateStorageStatusHandlingShould extends AggregateStorageVisibilityHandlingShould {
+public class DsAggregateStorageStatusHandlingShould
+        extends AggregateStorageVisibilityHandlingShould {
```
Comment (dmdashenkov) [inline gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageStatusHandlingShould.java]: Please make `class` and `extends` keywords start at the same left margin.

**Review by dmdashenkov — APPROVED** [review]
@alexander-aleksandrov, LGTM except for two minor issues.

**Pull request merged**